buildroot/bin/build_example

```diff
@@ -27,8 +33,20 @@ for sub in find $2/config/examples/$3 -type d; do
   cp "$sub"/_Bootscreen.h       Marlin/ 2>/dev/null
   cp "$sub"/_Statusscreen.h     Marlin/ 2>/dev/null
 
+  # Shamelessly copied from mftest script, skimmed the dependencies
+  MB=$( grep -E "^\s*#define MOTHERBOARD" Marlin/Configuration.h | awk '{ print $3 }' | $SED 's/BOARD_//' )
+  [[ -z $MB ]] && { echo "Error - Can't read MOTHERBOARD setting." ; exit 2 ; }
+  BLINE=$( grep -E "define\s+BOARD_$MB\b" Marlin/src/core/boards.h )
+  BNUM=$( $SED -E 's/^.+BOARD_[^ ]+ +([0-9]+).+$/\1/' <<<"$BLINE" )
+  BDESC=$( $SED -E 's/^.+\/\/ *(.+)$/\1/' <<<"$BLINE" )
+  [[ -z $BNUM ]] && { echo "Error - Can't find $MB in boards list." ; exit 2 ; }
+  ENVS=( $( grep -EA1 "MB\(.*\b$MB\b.*\)" Marlin/src/pins/pins.h | grep -E "#include.+//.+(env|$SYS):[^ ]+" | grep -oE "(env|$SYS):[^ ]+" | $SED -E "s/(env|$SYS)://" ) )
+  [[ -z $ENVS ]] && { errout "Error - Can't find target(s) for $MB ($BNUM)." ; exit 2 ; }
+  ECOUNT=${#ENVS[*]}
+
+  echo "Deduced env: $ENVS ($ECOUNT)"
   echo "Building the firmware now..."
-  echo "$HERE/mftest" -a || exit 1
```

Comment:
Oops, this 'echo' shouldn't be here. Last I checked this worked fine and didn't require inlining of its contents.

Reply:
If you run mftest this way, it says that it's missing mfinfo. I've moved mfinfo at the same place as mftest, and then it says that you must run mfinit. mfinit does something that modifying the current git repository so that you can't resume the script anymore. I'm not sure mftest is the right tool here.

Reply:
The `mfinfo` call is not needed, so I have removed it.